docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_procedure.md

```diff
@@ -24,29 +24,17 @@ Use the `CREATE PROCEDURE` statement to create a procedure in a database.
 
 ## Syntax
 
-<ul class="nav nav-tabs nav-tabs-yb">
-  <li>
-    <a href="#grammar" class="nav-link" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
-      <img src="/icons/file-lines.svg" alt="Grammar Icon">
-      Grammar
-    </a>
-  </li>
-  <li>
-    <a href="#diagram" class="nav-link active" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
-      <img src="/icons/diagram.svg" alt="Diagram Icon">
-      Diagram
-    </a>
-  </li>
-</ul>
-
-<div class="tab-content">
-  <div id="grammar" class="tab-pane fade" role="tabpanel" aria-labelledby="grammar-tab">
-  {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/create_procedure,arg_decl_with_dflt,arg_decl,subprogram_signature,unalterable_proc_attribute,lang_name,subprogram_implementation,sql_stmt_list,alterable_fn_and_proc_attribute.grammar.md" %}}
-  </div>
-  <div id="diagram" class="tab-pane fade show active" role="tabpanel" aria-labelledby="diagram-tab">
-  {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/create_procedure,arg_decl_with_dflt,arg_decl,subprogram_signature,unalterable_proc_attribute,lang_name,subprogram_implementation,sql_stmt_list,alterable_fn_and_proc_attribute.diagram.md" %}}
-  </div>
-</div>
+{{%ebnf%}}
+  create_procedure,
+  arg_decl_with_dflt,
+  arg_decl,
+  subprogram_signature,
+  unalterable_proc_attribute,
+  lang_name,
+  subprogram_implementation,
+  sql_stmt_list,
+  alterable_fn_and_proc_attribute
+{{%/ebnf%}}
 
```

Comment:
the Alter Sequence (ddl_alter_sequence.md) page has a link to Names and identifiers that is broken

Reply:
Good catch. This is a subtle problem. I'd earlier edited the _name_ syntax rule in the EBNF file to have this definition:

```
name = '<Text Literal as specified in [Names and identifiers](../../names-and-identifiers/)>' ;
```

This is very important. Our YSQL doc has _never_, until my most-recent PR(s), addressed the rules that a _name_ must meet. It's stricter than just _text literal_. There's a length limit of 63 characters and stuff about _common_ and _exotic_ names leading to a discussion about _quoted_ and _unquoted_ identifiers (like what are the legal characters and esp. the legal _first_ characters).

The link works fine when you view _grammar_diagrams.md_ in the Browser. But the relative link breaks when the _name_ rule is included in a _grammar/diagram pair_.

The _grammar/diagram_ pair in _ddl_alter_sequence.md_ is the _only one_ among all such files under the _syntax_resources_ directory that includes the _name_ rule. But you wouldn't expect to do this, 'cos it's such a common rule. Rather, we expected the reader to click on it when viewing a humanly maintained content _.md_ file and to be taken off to _grammar_diagrams.md_.

The easy fix was simply to remove the _name_ rule from the definition so that _alter_sequence,name,alter_sequence_options_ becomes simply _alter_sequence,alter_sequence_options_.

This whole business brought another bug to light. The _alter_sequence_ rule uses the _sequence_name_ rule. And this is defined as _qualified _name_, which is in turn defined as a period-separated list of one-or-more _names_. So notwithstanding the argument above, the _name_ rule should never have been included in the _grammar/diagram pair on the _alter sequence_ page!